File: 1-two-sum/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        for(int i = 0; i < nums.length; i++) {
            for(int j = 0; j < nums.length; j++) {
                if(i != j && nums[i] + nums[j] == target) {
                    int[] arr = new int[2];
                    arr[0] = i;
                    arr[1] = j;
                    return arr;
                }
            }
        }
        int[] arr = new int[2];
        arr[0] = 0;
        arr[1] = 0;
        return arr;
    }
}

File: 1-two-sum/two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    let obj = {};

    for(let i = 0; i < nums.length; i++) {
        let num = target - nums[i];
        if(obj[num] != undefined) {
            return [obj[num], i];
        }else {
            obj[nums[i]] = i;
        }
    }

    return [0,0];
};